Formula/rtmpdump.rb

```diff
@@ -1,10 +1,10 @@
 class Rtmpdump < Formula
   desc "Tool for downloading RTMP streaming media"
   homepage "https://rtmpdump.mplayerhq.hu"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/r/rtmpdump/rtmpdump_2.4+20151223.gitfa8646d.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/r/rtmpdump/rtmpdump_2.4%2b20151223.gitfa8646d.orig.tar.gz"
+  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/r/rtmpdump/rtmpdump_2.4+20151223.gitfa8646d.1.orig.tar.gz"
+  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/r/rtmpdump/rtmpdump_2.4%2b20151223.gitfa8646d.1.orig.tar.gz"
   version "2.4+20151223"
```

Comment:
Looks effectively like a Debian repack of the same commit point:

```
rtmpdump (2.4+20151223.gitfa8646d.1-1) unstable; urgency=medium

  * Reimport upstream snapshot.
  * debian/get-git-source.sh: Create archive from FETCH_HEAD as intended.
  * debian/control:
    - Bump Standards-Version.
    - Update Vcs-*.
  * debian/librtmp-dev.manpages: Install librtmp.3 manpage.
  * debian/rtmp{srv,suck}.1: Provide manpages for rtmpsrv and rtmpsuck.

 -- Sebastian Ramacher <sramacher@debian.org>  Wed, 27 Apr 2016 12:05:18 +0200
```
We don't even really need to carry it, but since it looks like they killed the original tarball it's probably safe to carry without a version change on our end.